### ISSUE TITLE
When two Scheduler clusters are bound to the same SeedPeer cluster, dfget download from source

### ISSUE THREAD

**Issue by LetFu** [dragonflyoss/Dragonfly2#1384]
### Bug report:

I started two clusters (e.g., A and B) with different securityDomain settings for the dfget-daemon nodes in each cluster, and each cluster bound to a Scheduler cluster. I also bound both Scheduler clusters to the same SeedPeer cluster.
I pulled the image for the first time in cluster A, at which point it could be download from SeedPeer. Then I pulling the same image in cluster B. This time the dfget-daemon does not pull through the previous SeedPeer cache, but directly download from source.
#### logs
The node and host prefixes correspond to the two clusters respectively. Both the seedpeer cluster and the scheduler cluster replica is 1.
I first pulled the image through host1, and then through node1. 
Since the manager's and redis's and harbor's address is on the public network, I changed the address of the manager and redis in the log to xx.xx.xx.xx. I hope this doesn't affect your view. And I removed some logs with the following command:
```bash
sed -i '/finished unary call with code OK/d' xxx.log
```
The logs are as follows:
[scheduler-node0.log](https://github.com/dragonflyoss/Dragonfly2/files/8899449/scheduler-node0.log)
[dfdaemon-node1.log](https://github.com/dragonflyoss/Dragonfly2/files/8899446/dfdaemon-node1.log)
[scheduler-host0.log](https://github.com/dragonflyoss/Dragonfly2/files/8899448/scheduler-host0.log)
[dfdaemon-host1.log](https://github.com/dragonflyoss/Dragonfly2/files/8899444/dfdaemon-host1.log)
[seed-peer-deployer.log](https://github.com/dragonflyoss/Dragonfly2/files/8899450/seed-peer-deployer.log)

### Expected behavior:

Cluster B can be downloaded through SeedPeer when it is done again, and not download from source.

### How to reproduce it:

Start clusters A and B with docker-compose, and set different securityDomain for dfget-daemon in cluster A and cluster B respectively.
Then download the image in cluster A. After the image is downloaded, download the same image in cluster B.
Configuration Related:
       Instead of getting the address of the scheduler via mananger, dfdaemon specifies in the configuration.

### Environment:
- Dragonfly version: v2.0.3
```bash
dragonflyoss/dfdaemon    v2.0.3     caa7ecf62a92   51.3MB
dragonflyoss/manager     v2.0.3     2c7ec0b598b6   99.8MB
dragonflyoss/scheduler   v2.0.3     33f098ec825f   69.1MB
```
- OS: centos7 
- Kernel (e.g. `uname -a`): Linux xxx 5.4.156-1.el7.elrepo.x86_64
- Others:

Thx.

**Comment by gaius-qi** [comment]
![image](https://user-images.githubusercontent.com/15955374/173726322-598e56a9-e537-439e-b275-3369ce8b2a07.png)

Since the seed peer did not return the end time, the scheduler deletes the peer because the calculated piece download time is too long. It was fixed and merged into v2.0.3 as a hotfix, refer to https://github.com/dragonflyoss/Dragonfly2/commit/3a10f1229e7e078f0cd5805562e6ef47ae63a0b8

![image](https://user-images.githubusercontent.com/15955374/173726839-dcf33399-e644-436a-8f0b-bbd0afcf5442.png)

**Comment by LetFu** [comment]
> ![image](https://user-images.githubusercontent.com/15955374/173726322-598e56a9-e537-439e-b275-3369ce8b2a07.png)
> 
> Since the seed peer did not return the end time, the scheduler deletes the peer because the calculated piece download time is too long. It was fixed and merged into v2.0.3 as a hotfix, refer to [3a10f12](https://github.com/dragonflyoss/Dragonfly2/commit/3a10f1229e7e078f0cd5805562e6ef47ae63a0b8)
> 
> ![image](https://user-images.githubusercontent.com/15955374/173726839-dcf33399-e644-436a-8f0b-bbd0afcf5442.png)

This problem was solved after I upgraded to the latest v2.0.3 image. Thx!
